src/components/AmortizationRepaymentSchedule.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { FunctionComponent, ReactNode, useState } from "react";
+import { Input } from "@components/Input";
+import {
+	RepaymentScheduleAmountType,
+	RepaymentScheduleDataPoint,
+	RepaymentScheduleGraph,
+} from "./RepaymentScheduleGraph";
+import { RepaymentScheduleTable } from "./RepaymentScheduleTable";
+import { Button } from "./Button";
+import { ColumnsProps } from "./Table";
+
+interface AmortizationRepaymentScheduleProps {
+	children?: ReactNode;
+}
+
+export const AmortizationRepaymentSchedule: FunctionComponent<
```

Comment:
we should receive information like principal, interest and duration (and/or others) and generate the schedule dynamically based on that information. this is just displaying static test data now. same with bulletloan

Reply:
I noticed we have separate tickets for that stuff so you can disregard this comment